api_v3/lib/types/KalturaUserEntryFilter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @package api
+ * @subpackage filters
+ */
+class KalturaUserEntryFilter extends KalturaRelatedFilter
```

Comment:
This class should be auto-generated and should extend KalturaUserEntryBaseFilter.